lucille-core/src/main/java/com/kmwllc/lucille/core/Runner.java

```diff
@@ -364,7 +365,7 @@ private static ConnectorResult runConnectorInternal(Config config, String runId,
       }
     } else {
       try {
-        ConnectorThread connectorThread = new ConnectorThread(connector, publisher);
+        ConnectorThread connectorThread = new ConnectorThread(connector, publisher, ThreadNameUtils.getThreadName("ConnectorThread"));
```

Comment:
minor: we probably don't need "Thread" in the thread name. Could just pass "Connector" to ThreadNameUtils.getThreadName. Same with "IndexerThread"

Reply:
resolved in [2044c9f](https://github.com/kmwtechnology/lucille/pull/172/commits/2044c9f47eab57317e613780caed80c0de5958fd)